src/config/old-set-parser.c

```diff
@@ -171,7 +171,7 @@ static int ssl_protocols_to_min_protocol(const char *ssl_protocols,
 					 const char **error_r)
 {
 	static const char *protocol_versions[] = {
-		"SSLv3", "TLSv1", "TLSv1.1", "TLSv1.2",
+		"SSLv3", "TLSv1", "TLSv1.1", "TLSv1.2", "TLSv1.3",
```

Comment:
This change alone is not enough to allow "TLSv1.3" only servers.

Reply:
Yes it doesn't add TLSv1.3 per se. I looked through the code, config and documentation for occurrences that did not include it and added it there.

Reply:
https://github.com/dovecot/core/pull/126